spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -84,6 +84,12 @@
 
   context 'coming from an "I did not receive my letter" link in a reminder email' do
```

Comment:
Note that this spec tests when the user received an email 14 days after requesting a letter, and the email has a specific link in it to request another letter. I think we should set up this feature test to match those conditions and make sure the user *doesn't* see the spam warning banner. I'm concerned because we've seen a bug around this while testing the 14 day reminder mechanism.

I agree with combining feature specs wherever possible overall, but the banner could be tested in a different spec, or add a new one if necessary.